### PR TITLE
netdata: Update to version 1.17.1

### PR DESCRIPTION
**Pull request by BKPepe**
Maintainer: me and @diizzyy 
Compile tested: Turris Omnia, mvebu (cortex-a9), OpenWrt master
Run tested: Turris Omnia, mvebu (cortex-a9), OpenWrt master

Description:
- Update to version 1.17.1.
Changelog:
https://github.com/netdata/netdata/releases/tag/v1.17.0
https://github.com/netdata/netdata/releases/tag/v1.17.1

Patches:
Patch 002 - Force Python3 is no longer applying, let's try to throw it
away to see if it is still necessary. It was added to this repository almost one year ago. If it will be required, let's send it to upstream first.

Patch 003 - Was backported from the master branch and it was included in
Netdata version 1.17.0.

### CODE REVIEW

File: admin/netdata/Makefile [hunk 1]
```diff
@@ -18,7 +18,7 @@ PKG_CPE_ID:=cpe:/a:my-netdata:netdata
 
 PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/netdata/netdata/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=94492108a6e24e8b39c011ae35ff6f50a848d816af396fdf2b44655cecd78672
+PKG_HASH:=032f9001e2b7f774989a200519feeafa3a4d9b7c9e8fb0cfa43e68a8da6ad1bf
 
 PKG_INSTALL:=1
```
Comment (neheb) [inline admin/netdata/Makefile]: Out of curiosity, does PKG_BUILD_PARALLEL help here?

**Comment by diizzyy** [comment]
LGTM

File: admin/netdata/Makefile (hunk 1 above)
Comment (BKPepe) [reply admin/netdata/Makefile]: Good question. AFAIK: It should, but to be sure, I'm going to test it multiple times during the weekend with full builds. (Reference: https://gitlab.labs.nic.cz/turris/turris-build/commit/7646aa98e86463ae0fd04e0ce5e1173685705d26 )

Comment (diizzyy) [reply admin/netdata/Makefile]: I'm not sure if it's the toolchain logic itself or make but I'm seeing weird random breakage compared to other distributions which makes me a bit hesitant to add that flag.

Comment (neheb) [reply admin/netdata/Makefile]: I'd add it with 0 to make it explicit.

**Pull request merged**